db/migrate/20230125185751_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :Name
+      t.string :photo
+      t.text :Bio
+      t.integer :PostsCounter
```

Comment:
- [ OPTIONAL ] I highly recommend to make the name here posts_counter to follow the best practices in name convensions so you can add a new migration and rename this column to be posts_counter here ✅😊🔥♻💯